src/constants/index.ts

```diff
@@ -970,6 +991,7 @@ export const TBTC_POOL_TOKENS = [TBTC_V2, ...BTC_POOL_V2_TOKENS]
 export const TBTC_UNDERLYING_POOL_TOKENS = [TBTC_V2, BTC_SWAP_V2_TOKEN]
 
 export const EVMOS_BTC_POOL_TOKENS = [WBTC, RENBTC]
+export const EVMOS_4_POOL_TOKENS = [DAI, FRAX, USDC, USDT]
```

Comment:
The order of the tokens should be DAI, USDC, USDT, FRAX

https://github.com/saddle-finance/saddle-contract/pull/583/files